### PR TITLE
there are two types of variant we may need to push, adding them both

### PR DESCRIPTION
**Pull request by danieltmiles**

### CODE REVIEW

File: shopify.go [hunk 1]
```diff
@@ -123,6 +123,8 @@ func (s *Shopify) DoRequest(verb string, uri *url.URL, creds *Credentials, paylo
 			// retry
 			return s.DoRequest(verb, uri, creds, payload, jsonStruct)
 		}
+		bodyBytes, _ := ioutil.ReadAll(resp.Body)
+		fmt.Printf("bad response code, body:\n%v\n", string(bodyBytes))
```
Comment (rathgara) [inline shopify.go]: take this fmt.Printf out?

Comment (dan9186) [inline shopify.go]: Or maybe returned as part of the error?

File: variants.go [hunk 2]
```diff
@@ -41,20 +41,40 @@ type Variant struct {
...
+
+	} else { // we are adjusting a quantity
+		j, err = json.Marshal(&VariantUpdate{
+			Id:                 id,
+			Quantity:           newQty,
+			OriginalQuantity:   oldQty,
+			QuantityAdjustment: adjustment,
+		})
+	}
+	fmt.Printf("new variant:\n%+v\n", string(j))
```
Comment (dan9186) [inline variants.go]: Probably don't want this print left in?

Comment (danieltmiles) [inline variants.go]: yeah, for the time being, I want these prints in there. In the fullness of time, I think we need to add a logger, silent components make me sad. :(

**Pull request merged**